### DISCUSSION TITLE
Is there an explanation or documentation of the Filter argument?

### DISCUSSION THREAD

**Discussion by MatthiasThoemel** [MarshalX/atproto#409] [Questions] [answered]
Some of the high level API calls understand a Filter argument, for example get_autor_feed. 

I cannot find any description of this Filter argument - can somebody point me to the documentation of it?

**Comment by MarshalX** [comment, marked as answer]
Possible values are: Literal['posts_with_replies'] | Literal['posts_no_replies'] | Literal['posts_with_media'] | Literal['posts_and_author_threads']

And default value is 'posts_with_replies'

Ref: https://atproto.blue/en/latest/atproto/atproto_client.models.app.bsky.feed.get_author_feed.html#atproto_client.models.app.bsky.feed.get_author_feed.Params.filter
